pkg/reconciler/reconcile.go

```diff
@@ -0,0 +1,22 @@
+package reconciler
+
+import (
+	"context"
+
+	"github.com/red-hat-storage/managed-fusion-fleet-reconciler/pkg/db"
+	"github.com/red-hat-storage/managed-fusion-fleet-reconciler/pkg/forman"
+
+	"go.uber.org/zap"
+)
+
+func Reconcile(logger *zap.Logger, dbInstance *db.Postgres, req forman.Request) forman.Result {
+	ctx := context.TODO()
```

Comment:
Let's use `context.Background()` and not `context.Todo()`. Or even better provide it from the outside